src/tool/subcommands/api_cmd.rs

```diff
@@ -1189,7 +1208,7 @@ async fn run_tests(
         FilterList::default().allow(config.filter.clone())
     };
 
-    for test in tests.into_iter() {
+    for test in tests.into_iter().unique_by(RpcTest::hashable) {
```

Comment:
nit: I'd add a comment here, and move the logic inline, or comment the RpcTest::hashable to explain what it's for

Reply:
Fixed.